dsgrid/project.py

```diff
@@ -183,32 +170,26 @@ def process_query(self, context: QueryContext, cached_project_mapped_datasets_di
         dim_columns = sorted(dim_columns)
         time_columns = sorted(time_columns)
         expected_columns = time_columns + pivoted_columns_sorted + dim_columns
```

Comment:
NIT: I think expected_columns should be `time_columns + dim_columns + pivoted_columns_sorted` to match ordering in DatasetExpressionHandler. I think this func here doesn't enforce final cols ordering anymore, it just makes things less confusing since DataExpressionHandler would otherwise change col ordering.